### DISCUSSION TITLE
Send request() commands to a End Device

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1132] [General]
I'm asking this question as I have a strange behaviour with zigpy_znp while I don't see it with the bellows one.

Basically when sending message to an end device not listening on idle, but working with polling, I have very often failure when doing the request() command.
This is especially happening when doing an OTA to an Ikea Remote, where the OTA often failed with that.
But also happen when sending a SetPoint to Danfoss eTRV for example

Is there anything special to do when send request() to an end-device polling based with zigpy_znp ?

**Comment by Hedda** [comment]
Could this now be achived via https://github.com/zigpy/zigpy/pull/1102 ?
